evaluation/digital_corpora_download.ipynb

```diff
@@ -48,41 +109,36 @@
     "    \"2399488\",\n",
     "]\n",
     "\n",
-    "if not os.path.exists(bo_20_path):\n",
-    "    os.makedirs(bo_20_path)\n",
-    "\n",
-    "def get_zip_range(zip_id):\n",
-    "    lower = floor(int(zip_id)/1000) * 1000\n",
-    "    upper = lower + 999\n",
-    "    return str(lower).zfill(4) + \"-\" + str(upper).zfill(4) + \"/\"\n",
-    "\n",
-    "\n",
-    "for pdf_id in pdf_ids:\n",
-    "    pdf_path = os.path.join(bo_20_path, pdf_id+\".pdf\")\n",
-    "    if not os.path.exists(pdf_path):\n",
-    "        print(f\"Downloading pdf: {pdf_id}.pdf. Note: this requires downloading a large zipfile so it may take a while\")\n",
-    "        base_url = \"https://corp.digitalcorpora.org/corpora/files/CC-MAIN-2021-31-PDF-UNTRUNCATED/zipfiles/\"\n",
-    "        zip_id = pdf_id[:4]\n",
-    "        zip_range = get_zip_range(zip_id)\n",
-    "        full_url = base_url + zip_range + zip_id + \".zip\"\n",
-    "\n",
-    "        if not os.path.exists(bo_20_path + \"temp\"):\n",
-    "            os.makedirs(bo_20_path + \"temp\")\n",
-    "        r = requests.get(full_url)\n",
-    "        z = zipfile.ZipFile(io.BytesIO(r.content))\n",
-    "        z.extractall(bo_20_path + \"temp\")\n",
+    "download_pdfs(bo20_path, bo20_ids)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6639aae6-de2f-4a22-bfa2-826750c92851",
+   "metadata": {},
+   "source": [
+    "## Bo 767"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "986f108f-eef3-4815-acb9-ba74cf781081",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bo767_path = \"../data/bo767/\"\n",
     "\n",
-    "        # Move desired file to bo_20 folder\n",
-    "        os.rename(bo_20_path + \"temp/\" + pdf_id + \".pdf\", bo_20_path + pdf_id + \".pdf\")\n",
+    "with open(\"bo767_ids.obj\", \"rb\") as f:\n",
```

Comment:
why do we use a binary object instead of just a static list in the notebook?

or if the list is long, why not a plain text file? Just hoping people can more easily inspect what's happening

Reply:
Changed it to a txt file